examples/trading-runner/README.md

```diff
@@ -0,0 +1,116 @@
+
+### 5/ Trade
+
+Once the "Trading" address has funds in it, execute the "trade" command to make a t
+
+```
+pnpm cli trade --baseAsset=<SYMBOL> --quoteAsset=<SYMBOL> --baseAmount=<WHOLE AMOUNT> --key=<USER>
+```
+
+Note: when trading ETH using Uniswap v2/v3, wrapping and unwrapping ETH/WETH will be handled under the hood.
+
+### 6/ Sweep
+
+Next, use the "sweep" command to move the assets from the "Trading" address(es) to the "Long Term Storage" address(es).
+
+```
+pnpm cli sweep --asset=<ASSET> --destination=<ADDRESS> --amount=<WHOLE AMOUNT> --key=<USER>
+```
+
+Notes:
+
+- User `bob` will be denied due to the policies if he attempts to send funds to an unauthorized destination.
+- If a `destination` is not specified, it will default to sending to the `Long Term Storage` address.
+- If an `amount` is not specified, it will default to sending the ~entire balance.
+
+## Sample trades
+
+```
+pnpm cli trade --baseAsset=ETH --quoteAsset=USDC --baseAmount=0.001 --key=bob # will auto-wrap ETH into WETH
+pnpm cli trade --baseAsset=WETH --quoteAsset=USDC --baseAmount=0.001 --key=bob
+pnpm cli trade --baseAsset=USDC --quoteAsset=WETH --baseAmount=1000 --key=bob
+pnpm cli trade --baseAsset=USDC --quoteAsset=ETH --baseAmount=1000 --key=bob # will auto-unwrap WETH into ETH
+```
+
+## Sample sweeps
+
+```
+pnpm cli sweep --asset=USDC --amount=1 --key=bob
+pnpm cli sweep --asset=USDC --amount=1 --key=bob --destination=0xf0609e87Dfa4DA10f38313868b15296f7B30c00A # will get denied
+```
+
+## Understanding policies
+
+First, see our [Policies docs](https://turnkey.readme.io/docs/policy-engine-overview) for a primer on how policies work and are written. You'll notice that the policies used in this demo make use of directly accessing the transaction data of the Ethereum transactions. For example, let's break down the transaction data for an ERC20 `transfer`, specifically USDC ([Etherscan link](https://goerli.etherscan.io/tx/0x11a4f4c0778ddbf7731cab1b07d7db577918397c47bf3270ea9016237c8d4d11)):
+
+```
+0xa9059cbb000000000000000000000000d3b433723858612da3260eac465758c7ddfa5e5000000000000000000000000000000000000000000000000000000000000f4240
+```
+
+The function signature is stored in the first 4 bytes after the `0x`, i.e. the first 8 hex characters. This evaluates to `a9059cbb`, and because our policy engine includes the `0x` prefix, this is why the policy checks the first ten characters (`eth.tx.data[0..10]`) to see if it equates to `0xa9059cbb`.
+
+Next, we have 2 static parameters, `to (address)` and `value (uint256)`, which each fit into 32 bytes, or 64 hex characters. Similar to accessing the `signature` chars, we can hone in on the `to` chars with `eth.tx.data[10..74]`, and `value` chars with `eth.tx.data[74..138]`. Note that these bits will not be 0x-prefixed, so we are comparing pure hex chars.
```

Comment:
let's mention that EVM call data parameters are are always 32 bytes (or 64 hex characters) and must be padded with 0s. Maybe an example would help too:

```
eth.tx.data[0..10]: "0xa9059cbb"
eth.tx.data[10..74]: "000000000000000000000000d3b433723858612da3260eac465758c7ddfa5e50"
eth.tx.data[74..138]: "00000000000000000000000000000000000000000000000000000000000f4240"
...
```


```